balancer-js/src/test/lib/utils.ts

```diff
@@ -137,3 +148,25 @@ export const updateBalances = async (
   }
   return Promise.all([bptBalance, ...balances]);
 };
+
+export const getErc20Balance = (
```

Comment:
Do you think we can get rid of tokenBalance function (or consolidate to use one helper function)?

Reply:
Yes, we should be able to consolidate, but I wasn't sure on how to proceed, since it requires refactoring on join integration tests that are already implemented on migrations branch.
I just checked and I made a single commit with that refactor on migrations branch, so I should be able to cherry pick it 👍